source/basic.tex

```diff
@@ -5262,8 +5261,8 @@
 \begin{note}
 A pointer past the end of an object\iref{expr.add}
 is not considered to point to an unrelated object
-of the object's type
-that might be located at that address.
+of the object's type,
+even if the unrelated object would be located at that address.
```

Comment:
Can we just say "is located" here? What does the extra qualification achieve here?

Reply:
fixed